src/vue/forms/SubmitTradeOfferForm.vue

```diff
@@ -194,4 +320,20 @@ export default {
       margin-top: 1.6rem;
     }
   }
+
+  .app__button-tooltip {
+    margin-left: 1.2rem;
+  }
+
+  .app__tool-tip {
+    padding: 1rem .5rem;
+    margin-top: 1rem;
+  }
+
+  .fade-enter-active, .fade-leave-active {
```

Comment:
BEMify the transition name (if it still stays)

Reply:
it doesn't matter it will be delete in next my commit but if u want i can